evm_loader/program/src/entrypoint.rs

```diff
@@ -527,21 +527,25 @@ fn process_instruction<'a>(
             let storage_info = next_account_info(account_info_iter)?;
 
             let operator_sol_info = next_account_info(account_info_iter)?;
+            let operator_eth_info = next_account_info(account_info_iter)?;
             let incinerator_info = next_account_info(account_info_iter)?;
```

Comment:
Caller token account should be mentioned here like user_eth_info or other instructions should be rebuilt.

Reply:
fixed